### PR TITLE
Add flag for testing logs in content comparison

### PR DESCRIPTION
**Pull request by zigaLuksic**
Always great to find another issue the same day you release a patch version

### CODE REVIEW

File: eogrow/utils/testing.py [hunk 1]
```diff
@@ -256,6 +256,7 @@ def run_config(
     *,
     output_folder_key: str | None = None,
     reset_output_folder: bool = True,
+    check_logs: bool = True,
```
Comment (mlubej) [inline eogrow/utils/testing.py:259]: the parameter description in the docs is missing

**Comment by mlubej** [comment]
Can you elaborate a bit on what was the issue? Is it bad that we always checked logs (since it was hard-coded)? Did this cause issues sometimes?

**Comment by zigaLuksic** [comment]
> Can you elaborate a bit on what was the issue? Is it bad that we always checked logs (since it was hard-coded)? Did this cause issues sometimes?

If a pipeline does not use EOWorkflows (like the one for merging vectors) then it produces an empty `finished.json` file. Checking pipeline logs fails because 'no executions finished'.

**Comment by mlubej** [comment]
I just started working on the other MR now and noticed this, thanks!

**Review by mlubej — APPROVED** [review]

**Pull request merged**